backend/curriculum_tracking/tests/test_AgileCard.py

```diff
@@ -688,3 +688,104 @@ def test_cancel_workshop_attendance(self):
+        self.assertEqual(self.project_2.content_item, self.card_2.content_item)
+
+        # Starting the projects (project & project_2) and asserting that they are in status IN_PROGRESS
+        self.project_1.start_time = timezone.now() - timedelta(days=5)
+        self.project_2.start_time = timezone.now() - timedelta(days=5)
+        self.project_1.save()
+        self.project_2.save()
+        self.assertEqual(AgileCard.derive_status_from_project(self.project_1), AgileCard.IN_PROGRESS)
+        self.assertEqual(AgileCard.derive_status_from_project(self.project_2), AgileCard.IN_PROGRESS)
+
+        # Setting a request for review and creating four different review times (No reviews done yet) for project_1
+        self.request_review_time = self.project_1.start_time + timedelta(1)
+        self.project_1.request_review(force_timestamp=self.request_review_time)
+        self.time_one = self.project_1.start_time - timedelta(days=6)
+        self.time_two = self.project_1.start_time + timedelta(days=4)
+        self.time_three = self.project_1.start_time + timedelta(days=3)
+        self.time_four = self.project_1.start_time + timedelta(days=2)
+
+    def test_get_users_that_reviewed_since_last_review_request_for_project(self):
+
+        # Four reviews are made with the four review times above (No reviews done on project_2)
+        review_1 = factories.RecruitProjectReviewFactory(
+            status=NOT_YET_COMPETENT,
+            recruit_project=self.project_1,
+            timestamp=self.time_one
+        )
+        review_1.timestamp = self.time_one
+        review_1.save()
+
+        review_2 = factories.RecruitProjectReviewFactory(
+            status=COMPETENT,
+            recruit_project=self.project_1,
+        )
+        review_2.timestamp = self.time_two
+        review_2.save()
+
+        review_3 = factories.RecruitProjectReviewFactory(
+            status=EXCELLENT,
+            recruit_project=self.project_1,
+        )
+        review_3.timestamp = self.time_three
+        review_3.save()
+
+        review_4 = factories.RecruitProjectReviewFactory(
+            status=NOT_YET_COMPETENT,
+            recruit_project=self.project_1,
+        )
+        review_4.timestamp = self.time_four
+        review_4.save()
+
+        result = self.card_1.get_users_that_reviewed_since_last_review_request()
+        ids_which_can_be_returned = [review_2.reviewer_user.id, review_3.reviewer_user.id, review_4.reviewer_user.id]
+
+        # Making sure that reviews were done on card and not on card_2, if card_2 returns reviews then our function is
+        # returning the wrong stuff and therefore it is not working as it should.
+        self.assertEqual(sorted(result), sorted(ids_which_can_be_returned))
+        assert len(self.card_1.get_users_that_reviewed_since_last_review_request()) > 0
+
+    def test_get_users_that_reviewed_since_last_review_request_for_project_2(self):
```

Comment:
what does project_2 mean? 
Please make sure that the test function names are meaningful. Do you mean  Remember that people who aren't you will need to read this. If you name your functions after variables then that's not the way to go. We need to know the intention of the code.